client/dashboard/index.js

```diff
@@ -19,6 +21,14 @@ export default class Dashboard extends Component {
 				<Header sections={ [ __( 'Dashboard', 'wc-admin' ) ] } />
 				<StorePerformance />
 				<Chart />
+				<div className="woocommerce-dashboard__columns">
+					<div className="woocommerce-dashboard__column">
+						<Card title={ __( 'Top Selling Products', 'wc-admin' ) } />
+					</div>
+					<div className="woocommerce-dashboard__column">
```

Comment:
Could you pass this className into the Card, or do you see having multiple cards in each column?

Reply:
I believe the notion is we will eventually have multiple cards in a column.